### PR TITLE
resubscribe on reconnect

### PR DESCRIPTION
**Pull request by koonpeng**
## What's new

Fixes things not being updated after a reconnect.

Can be tested by using chrome's debug console, in the network tab, set throttle to "offline", then wait for socketio to timeout, then set it back to no throttle.

## Self-checks

- [ ] I have prototyped this new feature (if necessary) on Figma 
- [ ] I'm familiar with and follow this [Typescript guideline](https://basarat.gitbook.io/typescript/styleguide)
- [ ] I added unit-tests for new components
- [ ] I tried testing edge cases
- [ ] I tested the behavior of the components that interact with the backend, with an e2e test

### CODE REVIEW

File: packages/dashboard/src/components/map-app.tsx [hunk 1]
```diff
@@ -310,7 +309,15 @@ export const MapApp = styled(
       const sub = rmf.fleetsObs
         .pipe(
           switchMap((fleets) =>
-            merge(...fleets.map((f) => (f.name ? rmf.getFleetStateObs(f.name) : EMPTY))),
+            merge(
+              ...fleets.map((f) =>
+                f.name
+                  ? rmf
+                      .getFleetStateObs(f.name)
+                      .pipe(throttleTime(5000, undefined, { leading: true, trailing: true }))
```
Comment (aaronchongth) [inline packages/dashboard/src/components/map-app.tsx:317]: Can we have the throttle time to be 1000?
5000 makes the robots rather glitchy and jump around every 5 seconds. The motion of the robot is quite important for remote monitoring so I think 1000 milliseconds is good, while every other subscription we can keep at 5000.

**Review by aaronchongth — CHANGES_REQUESTED** [review]
Thank you! Looks great, I only have 1 comment about the fleet subscription on map app

File: packages/dashboard/src/components/map-app.tsx (hunk 1 above)
Comment (koonpeng) [reply packages/dashboard/src/components/map-app.tsx]: reduced to 500 and remove unused code

**Comment by koonpeng** [comment]
Reduced throttle for other topics to 3s for more responsive ui.

**Review by aaronchongth — APPROVED** [review]
Awesome, thanks!
